components-rs/common.h

```diff
@@ -72,7 +72,13 @@ typedef struct ddog_Error {
  * so every operation on it is unsafe!
  */
 typedef struct ddog_Slice_CChar {
+  /**
+   * Must be non-null and suitably aligned for the underlying type.
+   */
   const char *ptr;
+  /**
+   * The number of elements (not bytes) that `.ptr` points to.
```

Comment:
Maybe in libdatadog I should modify this to say:

> The number of elements (not bytes, unless they are the same for a given type) that `.ptr` points to.

Seems misleading for `const char` ^_^